index.html

```diff
@@ -2,13 +2,74 @@
 <html lang="ko">
```

Comment:
KR로 국가를 명시해주어도 좋을 것 같습니다!
```suggestion
<html lang="ko-KR">
```